custom/icds_reports/ucr/data_sources/vhnd_form.json

```diff
@@ -595,6 +595,13 @@
     },
     "named_filters": {},
     "engine_id": "icds-ucr-citus",
+    "sql_column_indexes": [
```

Comment:
I am going to create the index on UCR table using concurrently prior to merging this using below command. Does this sound right?
```
CREATE INDEX CONCURRENTLY ix_ucr_icds-cas_static-vhnd_form_28e7fd58_ vhsnd_date_past_month ON ucr_icds-cas_static-vhnd_form_28e7fd58 (vhsnd_date_past_month);
```

Reply:
Cool, this is now created.